collectors/carbonblack/collector.js

```diff
@@ -25,16 +25,16 @@ let tsPaths = [];
 class CarbonblackCollector extends PawsCollector {
 
     constructor(context, creds) {
-        super(context, creds, packageJson.version);
+        super(context, creds, packageJson.version,process.env.paws_collector_param_string_1);
```

Comment:
I suggest we validate `paws_collector_param_string_1` before passing it to the constructor.

Reply:
Parsing the json  in al-aws-collector [checkin function ](https://github.com/alertlogic/al-aws-collector-js/pull/62/files#diff-c06f5f0e08865e628912333b6bb7941cf34fcc02b6fc4163a80e350b5bfd4c47R411)